zomes/chat/src/validation.rs

```diff
@@ -106,7 +117,7 @@ pub(crate) fn common_validatation(data: ValidateData) -> ExternResult<ValidateCa
                             Some(element_pkg) => {
                                 match element_pkg.signed_header().header() {
                                     Header::AgentValidationPkg(pkg) => {
-                                        return joining_code(pkg.author.clone(), pkg.membrane_proof.clone())
+                                        return joining_code(pkg.author.clone(), pkg.membrane_proof.clone(), holo_agent(zome_info()?.properties)?)
```

Comment:
Shouldn't you be using the `holo_agent` arg here instead?

Reply:
It is. The return of the holo_agent() is the `holo_agent` `AgentPubKey`